s32217-OskarNiedzwiedzki/pamietnik.txt

```diff
@@ -30,4 +30,11 @@ Poznałem także procesy i sposoby ich sprawdzania kontroli. Pomogła mi w tym k
 działających w tle. Zwieńczeniem dzisiejszych zajęć była komenda kill, która pozwala wysyłać różne sygnały do programów, w tym sygnał SIGKILL,
 który bezzwłocznie wyłączał program.
 
-Mam nadzieję, że dzięki tym wszystkim narzędziom będę mógł jeszcze efektywniej korzystać z linuxa
\ No newline at end of file
+Mam nadzieję, że dzięki tym wszystkim narzędziom będę mógł jeszcze efektywniej korzystać z linuxa
+
+09.12.2023
+Drogi pamiętniczku. To chyba już koniec. Ostatnim tematem było powtórzenie do kolokwium. Omówione zostały wszystkie dotychczasowe tematy.
+Prawda - piszę to już po zdaniu kolokwium. Ogólnie było łatwe, myślałem że będzie trudniejsze, ba nawet bałem się, że mogę nie podołać. 
+Kolosa by mi się udało zdać nawet bez rąk prawdopodobnie, dla utrudnienia i większych emocji rozważyłbym rozwiązanie tego kolosa z wyłączonym monitorem z nadzieją, że
+trafiłem w odpowiednią odpowiedź. No a tak to niestety było bez emocji, ale najważniejsze, że jest zdane
```

Comment:
Nie. Nie to jest najważiejsze.
Najważniejsza jest pozyskana wiedza i świadomość swoich własnych braków oraz tego czego powinno się jeszcze nauczyć. Jeśli wiesz ile jeszcze nie wiesz to już bardzo dużo wiesz.